manifests/02-appgate-operator.yaml

```diff
@@ -31,46 +31,46 @@ spec:
             - name: APPGATE_OPERATOR_HOST
               valueFrom:
                 configMapKeyRef:
-                  name: appgate-operator-conf
+                  name: appgate-sdp-configmap
```

Comment:
your naming is definitively better but could you update this as well https://github.com/appgate/sdp-operator/blob/master/manifests/01-config.yaml#L4 please.

Also maybe the references to that configmap in the README file.

Reply:
updated https://github.com/appgate/sdp-operator/blob/master/manifests/01-config.yaml#L4 and the README:

https://github.com/appgate/sdp-operator/pull/114/commits/e9c6f555bf1d1c871f766f148e8a81a4b9142c0e